build.xml

```diff
@@ -77,6 +98,22 @@
     </exec>
   </target>
 
+  <target name="test" depends="build" description="Run JUnit tests">
+    <junit printsummary="no" haltonfailure="yes" forkmode="once" fork="true"
+      dir="${basedir}" maxmemory="512m">
```

Comment:
Does running under default memory value cause problems?

Reply:
Doesn't seem to. I copied this from the adaptor build file, trying to keep things as similar as possible. If there's no reason for it, I can remove it.

Reply:
As similar as possible for the build files LGTM. Thanks.